cloud/scope/load_balancer_reconciler.go

```diff
@@ -166,9 +166,6 @@ func (s *ClusterScope) CreateLB(ctx context.Context, lb infrastructurev1beta2.Lo
 		return nil, nil, errors.New("control plane endpoint subnet not provided")
 	}
 
-	if len(controlPlaneEndpointSubnets) > 1 {
-		return nil, nil, errors.New("cannot have more than 1 control plane endpoint subnet")
-	}
```

Comment:
I don't follow this change exactly. Is this something we should also do for the NLB reconciler?

Reply:
so LB supports multiple subnets as well, so we should not have this validation. NLB does not, please do double check in the code to make sure.